packages/react-composites/src/composites/CallComposite/CallComposite.tsx

```diff
@@ -45,6 +48,12 @@ export interface CallCompositeProps extends BaseCompositeProps<CallCompositeIcon
    * Flags to enable/disable or customize UI elements of the {@link CallComposite}.
    */
   options?: CallCompositeOptions;
+
+  /* @conditional-compile-remove(rooms) */
+  /**
+   * Set this to enable/disable capacities for different roles
+   */
+  role?: Role;
```

Comment:
Is this just for testing and it will usually be obtained through a useSelector? This seems like it should be an adapter level property instead of a direct Composite prop

Reply:
We've discussed this for a while - In this beta version, we will merge provider into main and keep it in a provider way(a localization-like solution) and we will decide whether make it selector level once call sdk and rooms sdk decided to provide permission object in stateful layer